Reorganize the Contributor Workflow in RTD (#1293)
nautobot/nautobot: nautobot/docs/development/index.md

- Jathan McCollum
- Jason Edelman

## Project Structure

Nautobot components are arranged into functional subsections called _apps_ (a carryover from Django vernacular). Each app holds the models, views, and templates relevant to a particular function:

- `circuits`: Communications circuits and providers (not to be confused with power circuits)
- `dcim`: Datacenter infrastructure management (sites, racks, and devices)
- `extras`: Additional features not considered part of the core data model
- `ipam`: IP address management (VRFs, prefixes, IP addresses, and VLANs)
- `tenancy`: Tenants (such as customers) to which Nautobot objects may be assigned
- `users`: Authentication and user preferences
- `utilities`: Resources which are not user-facing (extendable classes, etc.)

> - `virtualization`: Virtual machines and clusters
> ## Release Management

Reviewer: ```suggestion
- `virtualization`: Virtual machines and clusters

## Release Management
```

  - The final notification will be provided 3-5 days before the release drops.
- The dates and notifications will occur by updating the GitHub Release Milestone and on Slack.

### Long Term Support (LTS)

> The core team is currently evaluating publishing Long Term Support (LTS) version of Nautobot. At this time there is no formal target for this initial release. Our goal is to collect feedback from installations of Nautobot to help identify a maintainable and reliable LTS model. If you have interest in deploying an LTS version of Nautobot, or useful information to help inform the final LTS model, please contribute to the [GitHub Discussion thread](https://github.com/nautobot/nautobot/discussions/1291) around LTS.

Reviewer: ```suggestion
The core team is currently evaluating the possibility of publishing a Long Term Support (LTS) version of Nautobot. At this time there is no formal target for this initial release. Our goal is to collect feedback from users of Nautobot to help identify a maintainable and reliable LTS model. If you have interest in deploying an LTS version of Nautobot, or useful information to help inform the final LTS model, please contribute to the [GitHub Discussion thread](https://github.com/nautobot/nautobot/discussions/1291) around LTS.
```

  feature request. These ideas are what will be in scope to review when moving work from _Future_ to _Near Term_ as stated in the previous section.
- **Q&A** - Request help with installing or using Nautobot.

## Contributing

> We welcome many forms of contributions to Nautobot. While we understand most contributions will commonly come from developers, we encourage others to contribute in the form of docs, tutorials, and user guides. If you have other ideas for contributing, don't hesitate to open an issue or have a discussion in one of the forums below.

Reviewer: ```suggestion
We welcome many forms of contributions to Nautobot. While we understand most contributions will commonly come from developers, we encourage others to contribute in the form of docs, tutorials, and user guides. If you have other ideas for contributing, don't hesitate to open an issue or have a discussion in one of the forums above.
```

Author: Good catch!

  work is required to resolve them. It may take some time for someone to address
  your issue.

Bugs will follow our published workflow from inbound triage to ultimate terminal state, whether accepted or closed:

> ![Bug Workflow State Diagram](imgs/bug-wf.jpg)

Reviewer: Instead of creating another new directory for doc images (we already have `docs/img/`, `docs/media/` and `docs/user-guides/images/`) can these images be added to the existing `docs/media/development/` directory instead?

Author: Moved!
